tmail-backend/jmap/extensions-api/src/main/java/com/linagora/tmail/james/jmap/CalendarEventRepository.java

```diff
@@ -0,0 +1,30 @@
+/********************************************************************
+ *  As a subpart of Twake Mail, this file is edited by Linagora.    *
+ *                                                                  *
+ *  https://twake-mail.com/                                         *
+ *  https://linagora.com                                            *
+ *                                                                  *
+ *  This file is subject to The Affero Gnu Public License           *
+ *  version 3.                                                      *
+ *                                                                  *
+ *  https://www.gnu.org/licenses/agpl-3.0.en.html                   *
+ *                                                                  *
+ *  This program is distributed in the hope that it will be         *
+ *  useful, but WITHOUT ANY WARRANTY; without even the implied      *
+ *  warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR         *
+ *  PURPOSE. See the GNU Affero General Public License for          *
+ *  more details.                                                   *
+ ********************************************************************/
+
+package com.linagora.tmail.james.jmap;
+
+import java.time.ZonedDateTime;
+
+import org.apache.james.core.Username;
+import org.reactivestreams.Publisher;
+
+public interface CalendarEventRepository extends EventAttendanceRepository {
```

Comment:
I hesitated about whether to rename `EventAttendanceRepository` to `CalendarEventRepository`, or instead, keep `EventAttendanceRepository` (dedicated to attendance status logic) and create a new `CalendarEventRepository` that extends it.

And I a bit tend slightly towards the second approach.